tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_cards_in_cards_views.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.search.app import Search
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBrowserCardsInCardsView(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        self.test_url = self.marionette.absolute_url('mozilla.html')
+        self.test_url2 = self.marionette.absolute_url('mozilla_community.html')
+
+    def test_browser_cards_in_cards_views(self):
+        search = Search(self.marionette)
+        search.launch()
+        browser = search.go_to_url(self.test_url)
+        browser.wait_for_page_to_load()
+
+        browser.tap_menu_button()
+        search2 = browser.open_new_window()
+        search2.wait_to_be_displayed()
+        browser2 = search2.go_to_url(self.test_url2)
+        browser2.wait_for_page_to_load()
```

Comment:
I know we've been doing these calls in the tests for long, but I wonder if the test wouldn't be simpler with:
```py
browser2 = search2.go_to_url(self.test_url2)
self.assertEqual('Mozilla Community', browser2.title)
```
```py
@property
def title(self):
    self.wait_for_page_to_load()
    self.switch_to_content()
    title = self.marionette.title
    self.switch_to_chrome()
    return title
```